pkg/operator/k8sutil/crud.go

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
```

Comment:
```suggestion
Copyright 2019 The Rook Authors. All rights reserved.
```

Reply:
This is another instance of moving code from one file to another and then adding more stuff. I think we decided to keep the oldest copyright.